text_test.go

```diff
@@ -11,7 +11,7 @@ func TestName(t *testing.T) {
 
 func TestNameFail(t *testing.T) {
 	name := name()
-	if name != "Leande" {
+	if name != "Leander" {
```

Comment:
This looks quite stupid.

Reply:
```suggestion
	if name != "Leander" {
```
Make a variable for what is expected then compare.